src/main/java/com/google/sps/data/Constants.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.data;
+
+public final class Constants {
+    private Constants() {}
```

Comment:
move the constructor to the bottom of the class since it doesn't contain real logic.